proto/internalapi/sensor/image_iservice.proto

```diff
@@ -8,8 +8,9 @@ import "storage/image.proto";
 package sensor;
 
 message GetImageRequest {
-    storage.ContainerImage image = 1;
-    bool scan_inline             = 2;
+    storage.ContainerImage  image = 1;
+    bool scan_inline              = 2;
+    storage.Deployment deployment = 3;
```

Comment:
is the entire deployment needed or just specific parts? This is a pretty large proto, potentially, and the image (tag 1) is already part of it, so it's as if we are sending that bit of information twice

Reply:
Immediately whats needed is the namespace, in the next release I plan to use pull secrets as well, past that unknown.  If its preferred we only send the fields that are needed at a given time I can make that change. Lemme know.

Reply:
Changed to only pass namespace